src/cli/cli_mac_filter.cpp

```diff
@@ -341,6 +420,15 @@ template <> otError MacFilter::Process<Cmd("rss")>(Arg aArgs[])
             SuccessOrExit(error = aArgs[1].ParseAsHexString(extAddr.m8));
             otLinkFilterRemoveRssIn(GetInstancePtr(), &extAddr);
         }
+        /**
+         * @cli macfilter rss clear
```

Comment:
Can you move this block down one line just before 

`else if (aArgs[0] == "clear")`

And unindent it to align with the else if